lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -253,6 +253,9 @@ instance PassphraseMinLength "encryption-new" where
     passphraseMinLength _ = passphraseMinLength (Proxy @"encryption")
 instance PassphraseMaxLength "encryption-new" where
     passphraseMaxLength _ = passphraseMaxLength (Proxy @"encryption")
+instance PassphraseMinLength "seed" where passphraseMinLength _ = 16
+instance PassphraseMaxLength "seed" where passphraseMaxLength _ = 255
```

Comment:
I'd sleep better knowing that we aren't adding these instances to be honest. The semantic is rather inadequate for `seed`

Also, keep in mind that the length here refers to the number of utf-8 characters which are often encoded on more than 1 byte. So it'll be really wrong to mix the two concepts.

Reply:
I removed that